### DISCUSSION TITLE
v0.3.0 release checklist

### DISCUSSION THREAD

**Discussion by genaris** [openwallet-foundation/credo-ts#1006] [Announcements]
In a similar fashion than in #622, I'm proposing to summarize the issues/PRs we want to complete before making the 0.3.0 release.

I'm listing initially what's marked in the corresponding milestone (+ AIP 2.0 base support, my personal desire :stuck_out_tongue_closed_eyes:). But we can move forward some of them, or include anything missing that should be done before.

## 0.3.0

- [x] https://github.com/hyperledger/aries-framework-javascript/issues/495 (@genaris)
  - PR: https://github.com/hyperledger/aries-framework-javascript/pull/991
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/497  (@genaris) 
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/755 
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/859 (@morrieinmaas)
  - PR: https://github.com/hyperledger/aries-framework-javascript/pull/900
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/860 (@TimoGlastra)
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/923
  - PR: https://github.com/hyperledger/aries-framework-javascript/pull/935
  - PR: https://github.com/hyperledger/aries-framework-javascript/pull/1035
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/944 (@morrieinmaas)
  - PR:  https://github.com/hyperledger/aries-framework-javascript/pull/965
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/948
	- #1035 
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/955 (@TimoGlastra)
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/996 (@TimoGlastra)
  - PR: https://github.com/hyperledger/aries-framework-javascript/pull/1020
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/997

## Will be moved to 0.3.x

- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/356 (@NB-MikeRichardson)
   - PR: https://github.com/hyperledger/aries-framework-javascript/pull/795
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/496 
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/354 (@NB-PrasadKatkar)
  - PR: https://github.com/hyperledger/aries-framework-javascript/pull/760
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/917

Milestone: https://github.com/hyperledger/aries-framework-javascript/milestone/5

**Comment by TimoGlastra** [comment]
In terms of required/non-required my list would be like this:

## Required

- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/755 
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/923
  - PR: https://github.com/hyperledger/aries-framework-javascript/pull/935
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/948
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/955
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/944
  - PR:  https://github.com/hyperledger/aries-framework-javascript/pull/965
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/860
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/996
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/997

## Not Required

- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/356 (@NB-MikeRichardson)
   - PR: https://github.com/hyperledger/aries-framework-javascript/pull/795
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/497  (@genaris)
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/496 
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/354
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/917
- [x] https://github.com/hyperledger/aries-framework-javascript/issues/859
  - PR: https://github.com/hyperledger/aries-framework-javascript/pull/900
- [ ] https://github.com/hyperledger/aries-framework-javascript/issues/495 (@genaris)
  - PR: https://github.com/hyperledger/aries-framework-javascript/pull/991

**Comment by genaris** [comment]
I've updated checklist based on @TimoGlastra comment and what's already merged. Not completely sure about credentials work, so please don't hesitate to correct it if needed.

**Comment by JamesKEbert** [comment]
Also, we might want to consider updating the dependencies for indy-sdk-react-native with this release once https://github.com/hyperledger/indy-sdk-react-native/pull/56 is merged/released (as the installation process changed for iOS).
